api/observatory/abstract/__init__.py

```diff
@@ -0,0 +1,38 @@
+from astropy.time import Time
+
+from abc import ABC
+from typing import Set
+
+from common.minimodel import Observation, ObservationMode, Resource, Site
+
+
+class ObservatoryProperties(ABC):
```

Comment:
I was thinking about that as well and it seems like it might be a good idea to move site and instrument information here, since they are observatory-specific. Right now, I don't think we should worry about it too much yet but I just wanted to have this set up looking ahead for the future.